data/LinuxGamer.json

```diff
@@ -19,6 +20,11 @@
       "url": "https://twitter.com/the_linuxgamer",
       "icon": "twitter"
     },
+    {
+      "name": "Follow my Mastodon",
+      "URL": "https://fosstodon.org/@linuxgamer",
+      "icon": "mastodon"
+     },
```

Comment:
```suggestion
    },
```

Reply:
Fixed this now, thanks for pointing it out, couldn't see what the check didn't like XD